packages/typescript/src/diagnostics/emit.ts

```diff
@@ -13,7 +14,7 @@ export function emitDiagnostic(
   ts: typeof import('typescript'),
   context: PluginContext,
   host: DiagnosticsHost,
-  diagnostic: import('typescript').Diagnostic
+  diagnostic: Diagnostic
```

Comment:
This was written before `import type` was a thing, so thanks for moving it over :)